### PR TITLE
KEYCLOAK-14176 updated license for account preview

### PR DESCRIPTION
**Pull request by edewit**

### CODE REVIEW

**Review by vmuzikar — COMMENTED** [review]
I'm afraid this might be a bit more difficult than we though. All the dependencies are now bundled to a single JS file. Is there any way how can we tell what dependencies are really included there? I think we even have some new deps added with the My Resource page implementation, so this should be reflected too.

We'll need to update the `locations` attributes because many of them are now pointing to non-existing JS files as they are now part of the one big file.

Also, the licenses have to be updated for [RH-SSO](https://github.com/keycloak/keycloak/tree/master/distribution/feature-packs/server-feature-pack/src/main/resources/licenses/rh-sso) too.

**Comment by edewit** [comment]
All the dependencies are mentioned in the `package.json` with snowpack we
removed a lot of dependencies in favour of our own implementation. I guess
it would be better to write a little script to do this. I meant to ask
about this, why does it sometimes mention the individual files in the
folder and sometimes the entire package? Guess it will be easier and more
"robust" to just name the folders.

**Comment by edewit** [comment]
@vmuzikar update with the generated xml

**Comment by edewit** [comment]
@vmuzikar updated now that #7128 has been merged

**Comment by ssilvert** [comment]
@vmuzikar Can you take one last look before we merge?

**Comment by vmuzikar** [comment]
@ssilvert Sure, will re-review tomorrow.

**Comment by ssilvert** [comment]
> @ssilvert Sure, will re-review tomorrow.

@vmuzikar Thanks.  I'll try not to jump the gun this time.

**Review by vmuzikar — CHANGES_REQUESTED** [review]
Thanks for the update. Even though it's a bit hard to double check if are dependencies are correct, I know you're using some kind of snowpack plugin you've developed (good work, BTW!) to get the deps list, so there's not much room for error. BTW, we should [document](https://github.com/keycloak/keycloak/blob/master/docs/dependency-license-information.md) that, but that's out of scope of this PR.

Last thing to do is to run [download-license-files.sh](https://github.com/keycloak/keycloak/blob/master/distribution/licenses-common/download-license-files.sh).

File: distribution/feature-packs/server-feature-pack/src/main/resources/licenses/keycloak/licenses.xml [hunk 1]
```diff
@@ -861,214 +861,422 @@
...
+        <license>
+          <name>BSD-3-Clause</name>
+          <url>https://raw.githubusercontent.com/mridgway/hoist-non-react-statics/master/LICENSE</url>
+        </license>
+      </licenses>
+    </other>
+    <other>
+      <description>object-assign</description>
+      <locations>
+        <file>themes/keycloak/common/resources/web_modules/common/index-fd2ed651.js</file>
```
[the remark this answers is not in the document]
Comment (edewit) [reply distribution/feature-packs/server-feature-pack/src/main/resources/licenses/keycloak/licenses.xml:1274]: > Last thing to do is to run [download-license-files.sh](https://github.com/keycloak/keycloak/blob/master/distribution/licenses-common/download-license-files.sh).

I already did that...

[the remark this answers is not in the document]
Comment (ssilvert) [reply distribution/feature-packs/server-feature-pack/src/main/resources/licenses/keycloak/licenses.xml:1274]: @vmuzikar Do you approve to merge this one now?

[the remark this answers is not in the document]
Comment (vmuzikar) [reply distribution/feature-packs/server-feature-pack/src/main/resources/licenses/keycloak/licenses.xml:1274]: We need to (re-)run the mentioned script. Currently, there are many license files missing. This is a problem for the licenses.html that is generated during the build and included in the distribution – there are many broken links to local files.

**Comment by edewit** [comment]
Updated all urls

**Review by vmuzikar — DISMISSED** [review]
LGTM, thanks @edewit!

Created a JIRA to document for others how to actually update the licenses: https://issues.redhat.com/browse/KEYCLOAK-14485

**Review by vmuzikar — APPROVED** [review]

**Review by ssilvert — APPROVED** [review]

**Pull request merged**